pytorch_lightning/utilities/params_tying.py

```diff
@@ -0,0 +1,70 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
naming nit: should this be `parameter_tying` too? that way everywhere the terminology is the same?